pyproject.toml

```diff
@@ -21,7 +21,7 @@ click = "^8"
 plumbum = "^1.7.0"
 Inject = "^4.3.1"
 structlog = "^21.1.0"
-PyYAML = "^5.4.1"
+PyYAML = "6.0.1"
```

Comment:
Is there a reason that we pin this specific version (not a range)?

Reply:
+1. I think this should be `PyYAML = "^6.0.1"`

Reply:
@ubatmongo

Reply:
Done